app/auth/views/fido.py

```diff
@@ -0,0 +1,111 @@
+import json
```

Comment:
Probably a nitpicking thing here but the imports are not sorted and sometimes unused imports are not removed :). The imports are usually sorted by first built-in libs, then external libs (i.e. those installed via `pip`) and lastly in-project modules. If you use Pycharm the "Optimize Imports" can do the job or https://github.com/timothycrosley/isort seems to be a good option.

Reply:
Hmm, I don't think VS Code has this feature built-in. Could you handle this instead?

Reply:
Yes I will run Pycharm auto-import tool after merging this PR.